src_py/sprite.py

```diff
@@ -1560,6 +1558,7 @@ def __repr__(self):
         """
         Turn the class into a string.
         """
+        # pylint: disable=consider-using-f-string
         return "<{klass} @{id:x} {attrs}>".format(
```

Comment:
Minor detail but, can we change this to use f-strings (and also change some similar code above in this file) for more consistency? To not clutter the f-string we can always use temporary variables outside the string :)

Reply:
I tried that, but it turned out looking like an evil monster.